packages/manager/modules/pci/src/projects/project/project.module.js

```diff
@@ -29,7 +29,6 @@ import privateRegistry from './private-registry';
 import storages from './storages';
 import users from './users';
 import vouchers from './vouchers';
-import regions from './regions';
```

Comment:
since you are removing this dependency are you sure that the "regions" module is not used ? are you importing the cloud-universe-component "regions" module elsewhere ?

Reply:
I will delete this line

The module 'cloud-universe-components' is already imported for guides